### PR TITLE
Ubuntu 20.04

### PR DESCRIPTION
**Pull request by FPoupon**
Fix errors on Ubuntu-20.04

### CODE REVIEW

File: bin/bv_packaging [hunk 1]
```diff
@@ -876,7 +876,7 @@ def create_debian_packages(options, selected_packages, dependencies):
         os.makedirs(output)
     # Looking for architecture using apt-config if possible
     architecture = re.search(r'APT::Architecture *\"([^\"]*)\";?',
-                             subprocess.Popen(('apt-config', 'dump'), stdout=subprocess.PIPE, stderr=subprocess.STDOUT).communicate()[0])
+                             str(subprocess.Popen(('apt-config', 'dump'), stdout=subprocess.PIPE, stderr=subprocess.STDOUT).communicate()[0],'UTF-8'))
```
Comment (denisri) [inline bin/bv_packaging:879]: `str(b'something', 'UTF-8') `only works in python3, not in python2
It can be replaced with `six.ensure_str(b'something')`

**Comment by denisri** [comment]
The proposed change would not work in python2, is it possible to replace it please ?

**Comment by FPoupon** [comment]
Ok, I'll do the change

**Comment by FPoupon** [comment]
Change done and checked on ubuntu-18.04/python2 and ubuntu-20.04/python3

**Pull request merged**